cost/aws/instance_cloudwatch_utilization/AWS Inefficient Instance Utilization using CloudWatch.pt

```diff
@@ -51,30 +58,29 @@ parameter "param_exclusion_tag_key" do
   allowed_pattern /([\w]?)+\:([\w]?)+/
 end
 
-parameter "param_take_action_tag" do
-  category "User Inputs"
-  label "Action Tag Key:Value"
-  description "Cloud native instance tag key and value that will be added to an instance identified for action. Example: downsize:true"
-  type "string"
-  allowed_pattern /([\w]?)+\:([\w]?)+/
-end
-
 ###############################################################################
 # Authentication
 ###############################################################################
 
-auth "auth_aws", type: "aws" do
-  version 4
-  service "ec2"
-  access_key cred('AWS_ACCESS_KEY_ID')
-  secret_key cred('AWS_SECRET_ACCESS_KEY')
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
+  label "AWS"
+  description "AWS IAM"
```

Comment:
please use this for the description: **Select the AWS Credential from the list.**

Reply:
I have updated the description.